bin/lib/barnier-filter.js

```diff
@@ -0,0 +1,8 @@
+const debug = require('debug')('bin:lib:barner-filter');
+
+const CANDIDATES = process.env.BARNIER_LIST || [];
```

Comment:
I don't think we should hard code values into our apps. We can parse the env var as JSON if you think that would be better/more readable?